src/ontology/cl-edit.owl

```diff
@@ -32633,6 +32636,14 @@ AnnotationAssertion(terms:date obo:CL_4047019 "2024-09-24T12:11:09Z"^^xsd:dateTi
 AnnotationAssertion(rdfs:label obo:CL_4047019 "early enterocyte"@en)
 EquivalentClasses(obo:CL_4047019 ObjectIntersectionOf(obo:CL_0000584 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0013740)))
 
+# Class: obo:CL_4047025 (type G cell of stomach)
+
+AnnotationAssertion(terms:contributor obo:CL_4047025 <https://orcid.org/0009-0005-7919-4905>)
+AnnotationAssertion(terms:date obo:CL_4047025 "2024-10-24T10:43:22Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4047025 "type G cell of stomach"@en)
+EquivalentClasses(obo:CL_4047025 ObjectUnionOf(ObjectIntersectionOf(obo:CL_0000508 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001276)) ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0004997)))
+SubClassOf(obo:CL_4047025 obo:CL_0000508)
```

Comment:
No need to add `SubclassOf type G enteroendocrine cell `. This is inferred.

Reply:
Removed on account of inference